hc2/headers/types/code_object_bundle.hpp

```diff
@@ -69,7 +69,7 @@ namespace hc2
                     std::copy_n(it, sizeof(y.cbuf), y.cbuf);
                     it += sizeof(y.cbuf);
 
-                    y.triple.insert(y.triple.cend(), it, it + y.triple_sz);
+                    y.triple.insert(y.triple.end(), it, it + y.triple_sz);
```

Comment:
@scchan if we really want to work around such things on our end, it is probably preferable to change the call to `y.triple.assign(it, it + y.triple_sz)`. For reference, the interface of assign is as follows: http://en.cppreference.com/w/cpp/container/vector/assign.

Reply:
but the insert is on `triple`, which is a `string` not a `vector`.  I can construct a new `string` and assign to triple

Reply:
That is OK @scchan, containers (string is confusingly sort of a container) have the same computational basis. See: http://en.cppreference.com/w/cpp/string/basic_string/assign.

Reply:
thanks, will submit a fix through a different branch/PR